CHANGELOG.md

```diff
@@ -27,6 +27,32 @@ Packages are formatted by release version. You can see all the changes done to p
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
+## [v12.0.0](https://github.com/shopsys/shopsys/compare/v11.1.0...v12.0.0) (2023-09-19)
+
+### :bug: Bug Fixes
+* [framework] fixed variant creation by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2801
+* [framework] fixed bestsellers edit in admin by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2799
+* [framework] fixed order edit error due to invalid type in vat object by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2800
+* [framework] fixed seoRobotsTxtContent null value in settings by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2731
+
+### :hammer: Developer experience and refactoring
+* [shopsys] removed deprecations before release 12.0 by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2719
+* [framework] upgraded doctrine/orm to latest version by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2774
+* [shopsys] updated overblog/graphql-bundle to stable version 1.0.0 with dependencies by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2788
+* [shopsys] updated Dockerfile to update installation of NodeJS and Postgres in PHP-FPM Dockerfile by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2792
+* [framework] encapsulation of AdditionalImageData by @pk16011990 in https://github.com/shopsys/shopsys/pull/1934
+* [shopsys] updated codeception to version 5 by @TomasLudvik https://github.com/shopsys/shopsys/pull/2611
+
+### :cloud: Infrastructure
+* [shopsys] improvements for deployment/running in cluster by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2609
+* [shopsys] CDN support by @grossmannmartin https://github.com/shopsys/shopsys/pull/2602
+
+### :placard: Other Changes
+* [coding-standards] updated slevomat coding standards by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2646
+* [shopsys] inheritdoc docblock is now unified and fixed automatically by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2612
+* [s3-bridge] fixed bundle conventions by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2645
+* [framework] fixed seoRobotsTxtContent null value in settings by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2731
```

Comment:
this is already mentioned in bug fixes section

Reply:
Removed